bahmni-emr/package/docker/Dockerfile

```diff
@@ -1,5 +1,8 @@
-FROM bahmni/openmrs:latest
+FROM bahmniindiadistro/openmrs:latest
 
-COPY bahmni-emr/resources/hipmodule*.omod ${OPENMRS_APPLICATION_DATA_DIRECTORY}/modules/
+COPY bahmni-emr/resources/openhmis.backboneforms*.omod ${OPENMRS_APPLICATION_DATA_DIRECTORY}/modules/
```

Comment:
@mohan-13 -- Right now these omods are static binaries checked into github. In future, i am guessing these will be "built" as part of the build process, and then pulled in here (latest ones). Is this understanding correct?